WordPress/UITestsFoundation/Screens/Editor/EditorPostSettings.swift

```diff
@@ -74,15 +73,15 @@ public class EditorPostSettings: BaseScreen {
             XCTAssertTrue(imageCount == 0, "Featured image is set but should not be")
         }
 
-        return EditorPostSettings()
+        return try EditorPostSettings()
     }
 
-    // returns void since return screen depends on which editor you're in
+    /// - Note: Returns `Void` because the return screen depends on which editor the user is in.
     public func closePostSettings() {
         navBackButton.tap()
     }
 
     public static func isLoaded() -> Bool {
-        return XCUIApplication().tables["SettingsTable"].exists
+        return (try? EditorPostSettings().isLoaded) ?? false
```

Comment:
Rather than duplicating the logic to verify if a certain element exists, generating code that risks getting out of date, we can leverage the `ScreenObject` `init` behavior, which oden't fail the tests automatically if the screen is not loaded.